tests/ml_models/test_mcorr_wavelet_pytest.py

```diff
@@ -0,0 +1,201 @@
+# SPDX-License-Identifier: Apache-2.0
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+# Any modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+"""
+Tests of correctness and compilation for the wavelet module
+of metrics correlation.
+"""
+
+import os
+
+import numpy as np
+import pywt
```

Comment:
we need to add this `pywt` in the `requirements-dev.txt` so that this plugin can be installed before testing. Otherwise integration test fails : https://github.com/opensearch-project/opensearch-py-ml/actions/runs/4548484040/jobs/8019564523?pr=126

```
 tests/ml_models/test_mcorr_wavelet_pytest.py:16: in <module>
    import pywt
E   ModuleNotFoundError: No module named 'pywt'
```